CONTRIBUTING.md

```diff
@@ -0,0 +1,110 @@
+**no contribution is too small.**
+
+
+## 💻Before Contributing
+
+Welcome to [arpitghura/typing-test](https://github.com/arpitghura/typing-test/). Before sending your pull requests, make sure that you **read the whole
+guidelines**. If you have any doubt on the contributing guide, please feel free to reach out.
+
+
+## 🙌Contribution
+
+Any contribution is accepted, from fixing grammatical mistakes to implementing complex scripts. Please read this section if you are contributing your work.
+
+
+### 🔖Steps to Contribute
+
+Following are the steps to guide you:
+
+* Step 1: Fork the repo and Go to your Git terminal and  clone it on your machine.
+* Step 2: Add a upstream link to main branch in your cloned repo
+    ```
+    git remote add upstream https://github.com/arpitghura/typing-test.git
+    ```
+* Step 3: Keep your cloned repo upto date by pulling from upstream (this will also avoid any merge conflicts while committing new changes)
+    ```
+    git pull upstream main https://github.com/arpitghura/typing-test.git
+    ```
+* Step 4: Create your feature branch (This is a necessary step, so don't skip it)
+    ```
+    git checkout -b <feature-name>
+    ```
+* Step 5: Commit all the changes (Write commit message as "Small Message")
+    ```
+    git commit -m "Write a meaningfull but small commit message"
+    ```
+* Step 6: Push the changes for review
+    ```
+    git push origin <branch-name>
+    ```
+* Step 7: Create a PR on Github. (Don't just hit the create a pull request button, you must write a PR message to clarify why and what are you contributing)
+
+
+### 🔨Note:
+
+> - Do not edit/delete someone else's script in this repository. You can only insert new scripts in this repository.
+
+  > - Give a meaningful name to whatever file or folder you are adding.
+  > - To state an example, folder name should be as "Images" and file name should be "front_image.jpg"
+
+
+## 🔑Guidelines
+
+1. Welcome to this repository, if you are here as open source program participant/contributor.
+2. Participants / contributors have to **comment** on issues they would like to work on, and mentors or the PA will assign you.
+3. Issues will be assigned on a **first-come, first-serve basis.**
+4. Participants / Contributors can also **open their issues** using [issue_template](https://github.com/arpitghura/typing-test/tree/main/.github/ISSUE_TEMPLATE),
+but it needs to be verified and labelled by a mentor or PA. Please discuss with the team once before opening your issues. We respect all your contributions, whether 
+it is an Issue or a Pull Request.
+6. When you raise a issue, make sure you get it assigned to you, before you start working on that project.
+7. Each participant / contributor will be **assigned 1 issue (max)** at a time to work.
+8. Participants are expected to follow **project guidelines** and [**coding style**](https://pep8.org/"). **Structured code** is one of our top priority.
```

Comment:
In `coding style` remove the pep8 link as it is the coding style defined for python but this project uses JavaScript, so Use Javascript Coding style link like 
- https://javascript.info/coding-style
- https://google.github.io/styleguide/jsguide.html